koala.gemspec

```diff
@@ -25,5 +25,5 @@ Gem::Specification.new do |gem|
 
   gem.add_runtime_dependency("faraday")
   gem.add_runtime_dependency("addressable")
-  gem.add_runtime_dependency("json", ">= 2.0")
+  gem.add_runtime_dependency("json", "~> 1.8")
```

Comment:
I'm actually going to update this to be >= 1.8 -- otherwise it wouldn't work with 2.0.